src/app/modules/data-set/annotation/annotation.component.html

```diff
@@ -1,6 +1,6 @@
 <form *ngIf="instanceId">
     <div class="container" style="flex-direction: column;">
-        <mat-form-field appearance="fill" floatLabel="always">
+        <mat-form-field appearance="fill" floatLabel="always" style="padding: 0px 5px 0px 5px;">
```

Comment:
Replace style with the class (it appears in several places in this file). This way, if we decide to change the padding of the form field, we can change it in one place.

.css:
```
.form-field {
    padding: 0px 5px 0px 5px;
}
```
.html:
`class="form-field"`